packages/contracts/src/Arbitrator.sol

```diff
@@ -54,19 +54,21 @@ contract Arbitrator is IArbitrator, Controlled(msg.sender, msg.sender), Toggleab
 
     /// @inheritdoc FeeCollector
     /// @dev Allows owner to update the arbitration fees.
-    function setFee(ERC20 token, uint256 amount) external override onlyOwner {
-        feeToken = token;
-        fee = amount;
+    function setFee(
+        ERC20 token,
+        address recipient,
+        uint256 amount
```

Comment:
Is this a _fixed_ fee?  Or will there be different fee amounts per dispute?

Reply:
For this initial version it will be a fixed fee for all disputes and appeals. We have also discussed the option of doing it proportional to the amount in dispute but without further feedback we decided to keep it simple.